Update example to v0.6.1

mkolesnik opened the pull request:
This should fix E2E

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

mkolesnik committed:
Update example to v0.6.1

This should fix E2E

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

mangelajo commented:
@mkolesnik cool checks: ERROR: Expected Shipyard version 0.6.0 but found 0.6.1 in lighthouse (releases/example.yaml)  may be we should go with v0.6.1 which was the final?

https://github.com/submariner-io/lighthouse/blob/v0.6.1/Dockerfile.dapper

mkolesnik committed:
Disable Admiral & Shipyard sync checks

Currently releases aren't automated so many of them fail these checks
which aren't that critical, disabling hem until we use a more automated
process where it will be enforced.

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

dfarrell07 reviewed (changes requested):
```
ERROR: Expected Admiral version v0.6.1 but found v0.6.0 in lighthouse (releases/example.yaml)
ERROR: Expected Admiral version v0.6.1 but found v0.6.0 in submariner (releases/example.yaml)
ERROR: Expected Shipyard version 0.6.1 but found 0.6.0 in admiral (releases/example.yaml)
```

mkolesnik commented:
As we discussed on the automation sync call, this also disables the sync checks for Admiral & Shipyard for the time being
